scripts/test-data/profile-l40s-x8.yaml

```diff
@@ -0,0 +1,136 @@
+chat:
```

Comment:
Just want to confirm this matches this file here? https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/nvidia/l40s/l40s_x8.yaml